packages/framework/aqueduct/package.json

```diff
@@ -103,6 +103,11 @@
   "module:es5": "es5/index.js",
   "typeValidation": {
     "version": "1.0.0",
-    "broken": {}
+    "broken": {
```

Comment:
Looks unrelated to this change.

Reply:
No it's from these type changes trickling up to this type. You can look at the old pipeline run for the error message if you're curious.